linter/lintfuncs.go

```diff
@@ -67,7 +87,30 @@ func LintThisUpdateDate(resp *ocsp.Response, leafCert *x509.Certificate) (LintSt
 		
 	}
 
-	return Passed, fmt.Sprintf("OCSP Response thisUpdate date %s for %s is within %s of the past", 
+	return Passed, fmt.Sprintf("OCSP Response thisUpdate date %s for %s is within %s in the past", 
 		resp.ThisUpdate, certType, DurationToString[thisUpdateLimit])
 	
 }
+
+// LintNextUpdateDate checks that an OCSP Response NextUpdate date is no more than NextUpdateLimitSubscriber in the past
+// Source: Apple Lint 04
+func LintNextUpdateDate(resp *ocsp.Response, leafCert *x509.Certificate) (LintStatus, string) {
+	if leafCert != nil && leafCert.IsCA {
+		return Passed, "OCSP Response nextUpdate lint not applicable to CA certificates"
+	}
+
+	limit, err := time.ParseDuration(NextUpdateLimitSubscriber)
+	if err != nil {
+		return Error, fmt.Sprintf("Could not parse time duration %s", NextUpdateLimitSubscriber)
+	}
+
+	if resp.NextUpdate.Sub(resp.ThisUpdate) > limit {
+		return Failed, fmt.Sprintf("OCSP Response NextUpdate date %s is more than %s after ThisUpdate date %s", 
+			resp.NextUpdate, DurationToString[NextUpdateLimitSubscriber], resp.ThisUpdate)
```

Comment:
Less important because they are in the same file, but consider doing a similar thing of all caps for other constants just so they are visible as constants `NextUpdateLimitSubscriber -> NEXT_UPDATE_LIMIT_SUBSCRIBER `